packages/dev/docs/pages/react-spectrum/theming.mdx

```diff
@@ -0,0 +1,107 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import docs from 'docs:@react-types/provider/src/index.d.ts';
+import {Layout, TypeLink} from '@react-spectrum/docs';
+export default Layout;
+
+```tsx import
+import {Provider} from '@react-spectrum/provider';
+import {ActionButton} from '@react-spectrum/button';
+```
+
+---
+category: Concepts
+---
+
+# Theming
+
```

Comment:
Is toast position part of theming? I guess not due to how we are implementing them as explained below.